source/features/previous-version.tsx

```diff
@@ -0,0 +1,128 @@
+import * as pageDetect from 'github-url-detection';
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+
+import delegate from 'delegate-it';
+
+import features from '../feature-manager';
+import * as api from '../github-helpers/api';
+import observe from '../helpers/selector-observer';
+
+import GitHubURL from '../github-helpers/github-url';
```

Comment:
```suggestion
import cache from 'webext-storage-cache';
import React from 'dom-chef';
import delegate from 'delegate-it';
import * as pageDetect from 'github-url-detection';

import features from '../feature-manager';
import observe from '../helpers/selector-observer';
import * as api from '../github-helpers/api';
import GitHubURL from '../github-helpers/github-url'; 
```

Nit pick just fixing the spacing

Reply:
Checked lint but GibHub Direct commit adds space on EOF.. 😞

Reply:
I don’t think that's a problem, we always use it so don't worry about it